LICENSES/vendor/github.com/kevinburke/go-bindata/LICENSE

```diff
@@ -0,0 +1,38 @@
+= vendor/github.com/kevinburke/go-bindata licensed under: =
+
+This work is subject to the CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
+license. Its contents can be found at:
```

Comment:
this is not in the CNCF list of allowed licenses... it looks like go-bindata was exempted, but not this new dep - https://github.com/cncf/foundation/blob/master/license-exceptions/cncf-exceptions-2019-11-01.json#L57-L61

Reply:
@liggitt are you referring to `github.com/dchest/safefile` as the new dep here?
`safefile` is released under the BSD 2-Clause "Simplified" License: https://github.com/dchest/safefile/blob/master/LICENSE - per https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist that is an allowed license.

Reply:
if you're referring to `github.com/kevinburke/go-bindata` itself as the new dep, please note that the currently vendored fork is `github.com/go-bindata/go-bindata` and thus already not the fork of go-bindata that has been explicitly exempted.